docs/eventing/brokers/broker-types/kafka-broker/README.md

```diff
@@ -257,6 +257,46 @@ spec:
 !!! note
     When using an external topic, the Knative Kafka Broker does not own the topic and is not responsible for managing the topic. This includes the topic lifecycle or its general validity. Other restrictions for general access to the topic may apply. See the documentation about using [Access Control Lists (ACLs)](https://kafka.apache.org/documentation/#security_authz).
 
+## Configure Kafka features
+
+There are various kafka features/default values the Knative Kafka Broker uses when interacting with Kafka. You can configure these as follows:
```

Comment:
We probably need to link to https://knative.dev/docs/eventing/experimental-features/ and clarify what each feature is about how each feature works, etc.
 
This seems overly generic for a user to grasp.

It might be worth considering a separate page that is linked to this one

Reply:
The term experimental is used wrongly in that doc, they are really just features, each at different maturity level (alpha, beta, or stable)